### PR TITLE
[TIP-854] Clean translations, step 1

### PR DESCRIPTION
**Pull request by pierallard**
# What did this PR ?

Today, in `master` branch, in the PIM, we have `3907` strings translations. It's a lot ! After some research, we found that there a lot of duplicate and unused strings. In fact, there is only `2349` strings once you removed duplicates.

So the first work to clean translations is to remove duplicate and some unused strings. That's why we introduced a `pim_common` key, containing all the common strings for all the PIM application, not related to a specific bundle.

This PR also introduced a lot of boyscoutism, by removing unused keys, by renaming hardcoded strings into real key paths, by reorganizing some parts of the translations.

It's only a first step, there is some work to finish it, but it begins to be huge so we stop this PR here and will continue with another one.

# Funny stats

- 66 occurences of "File validation"
- 20 occurences of "Cancel"
- 19 occurences of "Create"
- 17 occurences of "History"
- 16 occurences of "Confirm"
- 15 occurences of "Delete", .....

# Results

- In master: 3907 keys, 2349 without duplicates
- In this PR: 3452 keys, 2310 without duplicates

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

### CODE REVIEW

**Comment by tamarasaurus** [comment]
So if an integrator has overridden a label/translation key they might have to update that. Is that a common case ?

File: src/Oro/Bundle/DataGridBundle/Resources/translations/jsmessages.en_US.yml [hunk 1]
```diff
@@ -1,41 +0,0 @@
-"Item deleted": "Item deleted"
```
[the remark this answers is not in the document]
Comment (pierallard) [reply src/Oro/Bundle/DataGridBundle/Resources/translations/jsmessages.en_US.yml]: I looked for every key I dropped !

File: src/Pim/Bundle/ImportExportBundle/JobLabel/TranslatedLabelProvider.php [hunk 2]
```diff
@@ -1,71 +0,0 @@
-<?php
```
[the remark this answers is not in the document]
Comment (pierallard) [reply src/Pim/Bundle/ImportExportBundle/JobLabel/TranslatedLabelProvider.php]: Yes there was a weird shit with mix of front/back, now, everything is front.

File: src/Oro/Bundle/DataGridBundle/Resources/translations/jsmessages.en_US.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (tamarasaurus) [reply src/Oro/Bundle/DataGridBundle/Resources/translations/jsmessages.en_US.yml]: I believe you 👍

**Comment by pierallard** [comment]
@tamarasaurus Of course it will fail. I suppose it's not a common practice...

**Pull request merged**